### ISSUE TITLE
AWD LSTM dropout

### ISSUE THREAD

**Issue by sachinruk** [fastai/fastai#2296]
I might be completely out of my depth here but, was looking through the code and saw [this](https://github.com/fastai/fastai/blob/master/fastai/text/models/awd_lstm.py#L36): `F.dropout(w, p=self.weight_p, training=False`. With something similar in line 54.

Is there a point in using `F.dropout` with `training=False`. You might as well leave it out? Or did you mean to say `training=self.training`.

Happy to do a PR if I'm correct.

**Comment by sgugger** [comment]
No, this is just to properly initialize the weights with a copy of the raw weights. `training=False` is on purpose to have a copy of them. Note that in the forward function, we do use `training=self.training`, this is only for initialization.
